views/partials/eyes/eyes_1.erb

```diff
@@ -0,0 +1,2 @@
+<circle id="simple-r" fill="rgba(47, 57, 71, 1)" cx="113.8" cy="77" r="5.7"/>
```

Comment:
You technically don't need to make this `rgba` since it's full opacity. They could just be `rgb`